thorlcr/activities/aggregate/thaggregateslave.cpp

```diff
@@ -196,7 +196,6 @@ class AggregateSlaveActivity : public AggregateSlaveBase
     virtual void getMetaInfo(ThorDataLinkMetaInfo &info) const override
     {
         initMetaInfo(info);
-        info.singleRowOutput = true;
```

Comment:
removing excessive unused flags (not used anywhere), and can be determined by totalRowsMin==totalRowsMax==1